content/en/docs/concepts/security/secrets-good-practices.md

```diff
@@ -62,12 +62,16 @@ recommendations include:
 *  Implement audit rules that alert on specific events, such as concurrent
    reading of multiple Secrets by a single user
 
-#### Additional ServiceAccount annotations for Secret management
```

Comment:
Actually, in a page about _good practice_, maybe we should simply be recommending using namespaces as trust boundaries?

Reply:
Yes, IMO this section should be replaced with the following but I’m not sure how this impacts users who may have bookmarked the old section.

#### Restrict Access for Secrets

Use separate namespaces to isolate access to mounted secrets.

Reply:
PTAL